plugins/org.polymap.core.style/src/org/polymap/core/style/model/StyleRepository.java

```diff
@@ -59,37 +57,39 @@
 public class StyleRepository
         implements AutoCloseable {
 
-    private static Log log = LogFactory.getLog( StyleRepository.class );
-    
+    private static Log                          log        = LogFactory.getLog( StyleRepository.class );
+
     private EntityRepository                    repo;
-    
+
     private Cache<Pair<String,String>,Optional> serialized = CacheConfig.defaults().createCache();
 
-    
+
     /**
      * Creates a repository instance backed by a {@link LuceneRecordStore} in the
      * given dataDir.
      * 
      * @param dataDir The directory of the database or null, for testing in-memory.
-     * @throws IOException 
+     * @throws IOException
      */
     public StyleRepository( File dataDir ) throws IOException {
-        IRecordStore store = LuceneRecordStore.newConfiguration()
-                .indexDir.put( dataDir )
+        IRecordStore store = LuceneRecordStore.newConfiguration().indexDir.put( dataDir )
                 .create();
-        
-        repo = EntityRepository.newConfiguration()
-                .entities.set( new Class[] {
-                        FeatureStyle.class, 
-                        ConstantColor.class, 
-                        ConstantNumber.class,
-                        FilterMappedNumbers.class,
-                        //ScaleMappedNumbers.class,
-                        ConstantBoolean.class,
-                        ConstantFilter.class,
-                        PointStyle.class} )
-                .store.set( 
-                        new OptimisticLocking(
+
+        repo = EntityRepository.newConfiguration().entities.set( new Class[] {
```

Comment:
sorry, das ist mir erst beim Blick in den gepushden commit aufgefallen, dass alles formatiert ist. Ich dachte, ich hätte deine Formatierung auch in meinem eclipse eingestellt. Ich checke das noch mal. Diskutieren brauchen wir hier nicht.